src/app/(main)/leaderboard/UserPoints.tsx

```diff
@@ -24,24 +17,39 @@ const MyRank = styled(TableCell)`
   padding-right: 10px;
 `;
 
-const UserPoints: React.FC<UserDataProps> = ({
-  rank,
-  name,
-  connections,
-  points,
-  estimate,
-}) => {
+interface UserItem {
```

Comment:
Same, check if it's possible to use supabase's auto generated types

Reply:
imported